multiqc/modules/seqera_cli/seqera_cli.py

```diff
@@ -182,13 +204,29 @@ def _table(self, data_by_run):
             {v: scale.get_colour(i, lighten=0.5)} for i, v in enumerate(seqera_versions + nextflow_versions)
         ]
         repositories = list(set(d.get("repository") for d in data_by_run.values()))
+
+        def format_runUrl(x):
```

Comment:
I'd rather follow the Python convention to name functions and variables with lowercase letters and underscores only:
https://peps.python.org/pep-0008/#function-and-variable-names

I know there is a lot of differently styled older code, but we can keep in mind the preferred naming in all newly added code, and when refactoring :)

Reply:
Yes true - I did it like this because the JSON key is formatted that way. But agree that it's ugly. Feel free to fix 👍🏻